modular_zubbers/code/modules/jobs/job_types/clown.dm

```diff
@@ -0,0 +1,3 @@
+/datum/job/clown/New()
+	job_flags -= JOB_CANNOT_OPEN_SLOTS
+	return ..()
```

Comment:
Ok so I thought about this and this is a terrible way to do it if skyrat ever brings these back.

This however should work
```suggestion
/datum/job/clown/New()
	job_flags = job_flags & (~JOB_CANNOT_OPEN_SLOTS)
	return ..()
```